src/components/Footer.jsx

```diff
@@ -76,10 +76,10 @@ const Footer = props => (
         <Link to="/getting-started">Get Started</Link>
       </NavItem>
       <NavItem>
-        <Link to="/getting-started/#customize">Customize</Link>
+        <Link href="/guides">Guides</Link>
```

Comment:
The deep-link into the Getting Started page felt confusing and this kept the number of links in the navigation to 5.